cmd/kubeadm/app/master/apiclient.go

```diff
@@ -177,21 +177,22 @@ func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset, sched
 		n.ObjectMeta.Annotations[api.TaintsAnnotationKey] = string(taintsAnnotation)
 	}
 
-	if _, err := client.Nodes().Update(n); err != nil {
-		if apierrs.IsConflict(err) {
-			fmt.Println("<master/apiclient> temporarily unable to update master node metadata due to conflict (will retry)")
-			time.Sleep(apiCallRetryInterval)
-			attemptToUpdateMasterRoleLabelsAndTaints(client, schedulable)
+	for {
+		if _, err := client.Nodes().Update(n); err != nil {
+			if apierrs.IsConflict(err) {
```

Comment:
isn't this exactly what the `RetryOnConflict` is for? Also, how does ever succeed if you aren't changing `n`?